v-next/hardhat/src/internal/config-loading.ts

```diff
@@ -45,14 +45,14 @@ export async function resolveHardhatConfigPath(
  * @returns The absolute path to the closest Hardhat config file.
  * @throw HardhatError if no Hardhat config file can be found.
  */
-export async function findClosestHardhatConfig(): Promise<string> {
-  let hardhatConfigPath = await findUp("hardhat.config.ts");
+export async function findClosestHardhatConfig(from?: string): Promise<string> {
```

Comment:
This function is public, so we shouldn't change it's signature for testing, or change the export from `src/hre.ts` to be an alias that always passes `undefined` here.

If we do the alias, we should also have a docblock there.

Reply:
`findClosestHardhatConfig` is not exported from `src/hre.ts` and, other than `initHardhat` now, it was only ever used within the boundries of `config-loading.ts` - in `resolveHardhatConfigPath` to be exact. `resolveHardhatConfigPath` is also the function that's exported from `src/hre.ts`, btw. 

Do you think it would be OK to modify it then or should we leave it as is and maybe introduce a second function (something like `findClosestHardhatConfigFrom(from: string)`)?

Reply:
Agreed I think `resolveHardhatConfigPath` is in the public API, not `findClosestHardhatConfig`.

I think we leave the `from` addition rather than introducing a second function.

Reply:
oops, sorry, i misread the reexport 😅 @galargh